cypress/e2e/room_board/createEditDeleteFolderWithFileUploadInRoomBoard.feature

```diff
@@ -0,0 +1,68 @@
+@regression_test
+@stable_test
+Feature: Room Board - Create, edit and delete folder in board, including file handling inside the folder (in progress)
+
+    As a content editor I want to create a file folder in a room board
```

Comment:
can be a complete user story like -> ........, so that I can manage the files in the room.